etc/kayobe/users.yml

```diff
@@ -20,6 +21,15 @@ users_default:
   # Authorise all users' SSH keys for the kolla user. This allows them to
   # operate kolla.
   - "{{ lookup('file', kayobe_config_path ~ '/ssh-keys/id_rsa_bharat.pub') }}"
+  - "{{ lookup('file', kayobe_config_path ~ '/ssh-keys/id_rsa_isaac.pub') }}"
+
+- username: isaac
```

Comment:
Why the Isaac user here?

Reply:
Copying the convention for setting up users as per https://github.com/RSE-Cambridge/cumulus-kayobe-config/blob/rocky/etc/kayobe/users.yml

Presumably other users have been added manually here.